doc/tutorial/structure.rst

```diff
@@ -3,11 +3,108 @@
 
 .. _tutorial/structure:
 
-Custom studio file structure
-============================
+*********************
```

Comment:
@henriknorin-ftrack 

custom structure tutoral does not fit pipeline documentation.

Reply:
See my comment above regarding higher level docs/tutorial

Reply:
> Overall it looks good to me. But I would like to point out 2 things: 1: I think we should add a Disclaimer waring clients to not customise it or use it as test, because we will provide braking changes soon with the split of definitions + pipeline configs.... 2. The maya part where we encurage them to create a tools.py file in the maya integration, I think we shouldn't add it because the framework is thought to be modified in the definitions not in the integration, or in case we should provide the tools.py file which I'm not sure its a good idea.

Agree on the disclaimer, have added one but can add another one at a suitable location.

Regarding tools.py we need to first implement the utils definitions and plugins and move current DCC utils in there, that would then be the place were event driven tools can be called. The problem is that a tool is also borderline of a client and we need to revisit what a tool and a client is.

Reply:
@lluisFtrack  Please have a look at the existing disclaimer/note that are present in tutorial/introduction and see if does its job.

Reply:
@henriknorin-ftrack Can't find the Disclaimer, explaining the users that this code is totally subject to changes and should be customised only for testing proposes because it will break as soon as we split definitions or add pipeline configs.
Also, about the tools.py, I think we should not add that example, as the idea of the framework is that the user only modifies definitions, not the integration itself. But in any case, if you want to put that example, then I think it should go to utils.py as at least is a file that already exists and that we provide. But we don't provide the file tools.py.